docs/DevelopmentSetup/windows_installation.rst

```diff
@@ -46,3 +53,24 @@ Once this has completed, start a command shell in your FLINT repository folder.
 
     # OR Visual Studio Solution (2017)
     cmake -G "Visual Studio 15 2017" -DCMAKE_INSTALL_PREFIX=C:/Development/Software/moja -DVCPKG_TARGET_TRIPLET=x64-windows -DENABLE_TESTS=OFF -DENABLE_MOJA.MODULES.ZIPPER=OFF -DCMAKE_TOOLCHAIN_FILE=c:\Development\moja-global\vcpkg\scripts\buildsystems\vcpkg.cmake ..
+
+Running the project
+-------------------
+
+.. figure:: ../images/installation_vs2019_flint.example/Step4.png
+  :width: 600
+  :align: center
+  :alt: Alternative text
+
+  Running ``moja.cli.exe`` in Visual Studio Debugging All properties page
+
+We are running the ``moja.cli.exe`` from the moja.FLINT project here. In order to make edits to the Visual Studio Solution we can use the CMake GUI.
+
+**NOTE** : All paths used below with ``C:\Development\moja-global`` will need to be modified to match your system build location of the moja project.
+
+Edit solution using CMake GUI
+-----------------------------
+
+* Launch the CMake GUI
+* In the ``Where to build the binaries`` field click ``Browse Build…`` and select the folder you created above (i.e. ``C:\Development\moja-global\FLINT\Source\build``). The ``Where is the source code:`` field should update, if not, set it correctly.
```

Comment:
These instructions might also benefit from a screenshot so that it's clear where to look for each field.